ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource

```diff
@@ -343,3 +382,267 @@ Set RHODS Users Group To Inexistent Group
+        IF    "${tol_effect}" != "${EMPTY}"
+            Click Button    ${ACCELERATOR_TOLERATION_EFFECT_DROPDOWN}
+            Click Element    //label[@for="effect-select"]/parent::*/following::div//*[text()="${tol_effect}"]
+        END
+        IF    "${tol_key}" != "${EMPTY}"
+            Clear element and input text    ${ACCELERATOR_TOLERATION_KEY}    ${tol_key}
+        END
+        IF    "${tol_value}" != "${EMPTY}"
+            Clear element and input text    ${ACCELERATOR_TOLERATION_VALUE}    ${tol_value}
+        END
+        IF    "${tol_seconds}" != "${EMPTY}"
+            IF    "${tol_seconds}" != "Forever"
+                Click Element    ${ACCELERATOR_TOLERATION_SECONDS_CUSTOM_RADIO}
+                Clear element and input text    ${ACCELERATOR_TOLERATION_SECONDS_CUSTOM_VALUE}   ${tol_seconds}
+            ELSE
+                Click Element    ${ACCELERATOR_TOLERATION_SECONDS_FOREVER_RADIO}
+            END
+        END
+        Click Button    ${ACCELERATOR_TOLERATION_ADD_OR_UPDATE_BUTTON}
+    END
+    Click Button    ${CREATE_OR_UPDATE_ACCELERATOR_PROFILE_DETAIL_BTN}
+
+Delete Accelerator Profile Tolerations
+    [Documentation]    Delete Tolerarions from an Accelerator Profile and click on "Update acccelerator profile" button
+    ...                view: Edit Accelerator profile
+    [Arguments]   ${display_name}
+    Wait Until Element Is Visible    ${ACCELERATOR_EDIT_TOLERATION_KEBAB_BTN}
+    Click Element     ${ACCELERATOR_EDIT_NAV_MENU_TOLERATIONS}
+    ${tolerations_num}=  Get Element Count  ${ACCELERATOR_EDIT_TOLERATION_KEBAB_BTN}
+    FOR    ${counter}    IN RANGE    ${tolerations_num}
+        ${kebab_with_index}=    Catenate    SEPARATOR=    ${ACCELERATOR_EDIT_TOLERATION_KEBAB_BTN}    [1]
+        Click Button    ${kebab_with_index}
+        Wait Until Element Is Visible    ${kebab_with_index}/following::div//span[text()="Delete"]    timeout=10
+        Click Element    ${kebab_with_index}/following::div//span[text()="Delete"]
+    END
+    Click Button    ${CREATE_OR_UPDATE_ACCELERATOR_PROFILE_DETAIL_BTN}
+
+In The Accelerator Profiles Grid There Is An Accelerator Profile With Name
+    [Documentation]    Verify that there is an specific Accelerator Profile in the Grid
+    ...                view: Settings -> Accelerator profiles
+    [Arguments]    ${display_name}
+    Wait Until Element Is Visible   ${ACCELERATOR_PROFILE_GRID_COLUMN_NAME}//span[text()='${display_name}']  timeout=10
+
+Get Accelerator Profile Spec Values via CLI
+    [Documentation]  Get Spec fields from an specific Accelerator Profile Instance
+    [Arguments]   ${display_name}    ${ns}=redhat-ods-applications
+    ${ap_from_oc}=    Run     oc get acceleratorprofile -n ${ns} -o json | jq '.items[] | select(.spec.displayName == "${display_name}")'
+    ${ap_json}=    Load Json String    ${ap_from_oc}
+    ${ap_spec_dict}=    Get From Dictionary    ${ap_json}    spec
+    RETURN    ${ap_spec_dict}
+
+Can Not Get Accelerator Profile Via CLI
```

Comment:
do you mean that the profile should not be found when searched?
Then maybe it would be more clear to call this kw like `Accelerator Profile Should Not Exist` or `Accelerator Profile Should Not Be Found`. It's also a bit more RF style